.github/workflows/build-user-config.yml

```diff
@@ -68,7 +68,8 @@ jobs:
         run: |
           if [ -e zephyr/module.yml ]; then
             export zmk_load_arg=" -DZMK_EXTRA_MODULES='${GITHUB_WORKSPACE}'"
-            export new_tmp_dir=$(mktemp -d)
+            export new_tmp_dir="${TMPDIR:-/tmp}/zmk-config"
```

Comment:
Nit: I guess we don't need to `export` this variable, do we?

Reply:
Nope. Not needed. Fixed, and re-ran the workflow with no error.